NccsvConverter/NccsvConverter.NccsvParser/Helpers/NccsvParserMethods.cs

```diff
@@ -178,14 +191,122 @@ public static Variable CreateVariable(List<string[]> variableMetaData)
                 {
                     variableDataType = line[2];
                 }
+
+                if (line[1] == "*SCALAR*")
+                {
+                    variableDataType = GetTypeOf(line[2]);
+                }
             }
 
             variable.DataType = variableDataType;
 
-            return variable;
+        }
+
+        //when testing is done, this should maybe be made private?
+        public static string GetTypeOf(string value)
```

Comment:
good work! perhaps change "if" to "else if" when there's more than one if-statements in a row, for readability?